src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs

```diff
@@ -25,7 +25,7 @@ public InitialBuildProcess(IProcessExecutor processExecutor = null)
 
         public void InitialBuild(bool fullFramework, string projectPath, string solutionPath)
         {
-            _logger.LogInformation("Started initial build using {0}", fullFramework ? "msbuild.exe" : "dotnet build");
+            _logger.LogDebug("Started initial build using {0}", fullFramework ? "msbuild.exe" : "dotnet build");
```

Comment:
This means the user will be left wondering if stryker is doing anything, do we want that?

Reply:
No they won't! I added a new log somewhere else. Now the output looks like this:

```
[09:47:03 INF] The project C:\Dev\ExampleProjects\TestApp\TestApp\TestApp.csproj will be mutated
[09:47:05 INF] Building test project C:\Dev\ExampleProjects\TestApp\TestApp.NUnit\TestApp.NUnit.csproj (1/2)
[09:47:09 INF] Building test project C:\Dev\ExampleProjects\TestApp\TestApp.UnitTests\TestApp.UnitTests.csproj (2/2)
[09:47:18 INF] Using testrunner DotnetTest
```